test/Microsoft.AspNetCore.SignalR.Client.Tests/ConnectionTests.cs

```diff
@@ -389,7 +393,7 @@ public void CannotStartConnectionWithInvalidTransportType(TransportType requeste
 
             var closedTcs = new TaskCompletionSource<object>();
             var allowDisposeTcs = new TaskCompletionSource<object>();
-            int receivedInvocationCount = 0;
+            var receivedInvocationCount = 0;
 
             var connection = new HttpConnection(new Uri("http://fakeuri.org/"), new TestTransportFactory(mockTransport.Object), loggerFactory: null, httpMessageHandler: mockHttpHandler.Object);
             connection.Received +=
```

Comment:
allowDisposeTcs and receivedInvocationCount need to be removed because they are no longer needed.